README.md

```diff
@@ -248,6 +248,18 @@ module.exports = {
 }
 ```
 
+### Compile Packages
+
+Sometimes we have packages which need to be compiled through webpack along with our app. An great example of this is [seek-style-guide](https://github.com/seek-oss/seek-style-guide). Out of the box sku supports loading the seek-style-guide but if you need to treat other packages in this way you can use `compilePackages`.
```

Comment:
Docs should be written to address the user directly, rather than using words like "we". Also, in this case, it's probably important to frame the context of why you'd want to do this, e.g. "Sometimes you might want to extract and share code between sku projects, but this code is likely to rely on the same tooling and language features that this toolkit provides." etc.